migtests/tests/mysql/datatypes/validate

```diff
@@ -9,7 +9,7 @@ def main():
 #=============================================================================
 
 EXPECTED_ROW_COUNT = {
- 	'bit_types': 4,   
+ 	# 'bit_types': 4,   
```

Comment:
For each disabled "test/check", file a ticket and add the link in the comments here.

Fix all such occurrences.

Reply:
this is fixed by @makalaaneesh now